kie-server-parent/kie-server-tests/kie-server-integ-tests-all/src/test/java/org/kie/server/integrationtests/jbpm/cases/CaseSLAComplianceIntegrationTest.java

```diff
@@ -89,6 +90,7 @@ public void testStartCaseWithSLAEscalation() throws Exception{
         TaskSummary johnTask = johnTasks.get(0);
         assertThat(johnTasks).hasSize(1);
         assertThat(johnTask.getName()).isEqualTo("Hello1");
+        assertThat(johnTask.getProcessType()).isEqualTo(WorkflowProcess.CASE_TYPE);
```

Comment:
what about correlation key ?

Reply:
Now is testing that is not null not empty (so the conversion from TaskSummary model to TaskSummary service, that is what we are testing here, is properly perfomed)